File: sdkwork-backend-sdk-csharp/Models/PlusTenantAuditLogVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusTenantAuditLogVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? Id { get; set; }
        public int? TenantId { get; set; }
        public string? Action { get; set; }
        public string? Module { get; set; }
        public string? Operator { get; set; }
        public string? Description { get; set; }
        public string? Result { get; set; }
        public string? CreateTime { get; set; }
        public string? Ip { get; set; }
    }
}

File: sdkwork-backend-sdk-go/types/plus_tenant_audit_log_vo.go
package types

// 租户审计日志值对象
type PlusTenantAuditLogVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	TenantId int `json:"tenantId"`
	Action string `json:"action"`
	Module string `json:"module"`
	Operator string `json:"operator"`
	Description string `json:"description"`
	Result string `json:"result"`
	CreateTime string `json:"createTime"`
	Ip string `json:"ip"`
}
